pkg/user/api/validation/validation.go

```diff
@@ -10,16 +11,45 @@ import (
 
 func ValidateUserName(name string, prefix bool) (bool, string) {
```

Comment:
signature required for use with kvalidation.ValidateObjectMeta. Will name `_` to clarify intentional unuse